### PR TITLE
feat: New app `adapt-essentials-asset-fields`

### PR DESCRIPTION
**Pull request by zzzarius**
## Purpose

The common pattern for adding alt and description to the image is to upload the asset and reference it with dedicated wrapper content type. Which makes it a bit tedious for the editors.

The lean solution is to use title and description fields of the asset itself for the alt and description image attributes. But you can't impose any requirements that way. So it is easy to forget to fill in the title and description fields.

This app makes it easier to go through assets and edit it in bulk.

## Approach

Previously, when we needed to bulk update asset fields we used Contentful management SDK to pull all asset data into csv and push it back to Contentful. But it's error prone and values gets quickly outdated because content editors can't freeze their work.

## Testing steps

The app itself is really simple. Assets are listed with the title, description and status. You can edit the fields in place. The changes are saved automatically. You can also toggle additional fields or locale.

Once app is installed it can be accessed through app menu or set as homepage app.

## Breaking Changes

There are no breaking changes

## Dependencies and/or References

Original work is done in https://github.com/adaptdk/adapt-essentials-asset-fields
More info can be found in the [README.md](https://github.com/adaptdk/marketplace-partner-apps/blob/zzzarius/asset-fields/apps/adapt-essentials-asset-fields/README.md)

### CODE REVIEW

File: apps/adapt-essentials-asset-fields/package.json [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  "type": "module",
+  "scripts": {
+    "install-ci": "npm ci",
+    "dev": "vite",
+    "build": "tsc && vite build",
+    "test": "echo 'No tests yet'",
+    "lint": "eslint src --ext ts,tsx --report-unused-disable-directives --max-warnings 0",
+    "format": "prettier src/**/*.{ts,tsx} --write",
+    "preview": "vite preview",
+    "deploy": "contentful-app-scripts upload --ci --bundle-dir ./build --organization-id ${DEFINITIONS_ORG_ID} --definition-id 3Jt0c7RHcmPsB3mJAQC4iP --token ${CONTENTFUL_CMA_TOKEN}",
```
Comment (lilbitner) [inline apps/adapt-essentials-asset-fields/package.json:14]: do you mind changing the definition-id here to 6RGLVmXYrGaIIRIda1HwAC 
This is the new app definition that will exist within the Contentful Apps organization.

**Review by lilbitner — COMMENTED** [review]
Thanks for opening this PR! I have added just a few comments here. Let me know if you have any questions!

File: apps/adapt-essentials-asset-fields/package.json [hunk 2]
```diff
@@ -0,0 +1,47 @@
+{
+  "name": "adapt-essentials-asset-fields",
+  "private": true,
+  "version": "1.0.0",
```
[the remark this answers is not in the document]
Comment (zzzarius) [reply apps/adapt-essentials-asset-fields/package.json]: Sure. Fix https://github.com/contentful/marketplace-partner-apps/pull/661/commits/e676d638480a6d1a42c13abaf1c437dbe587a264

File: apps/adapt-essentials-asset-fields/package.json [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+  "scripts": {
+    "install-ci": "npm ci",
+    "dev": "vite",
+    "build": "tsc && vite build",
+    "test": "echo 'No tests yet'",
+    "lint": "eslint src --ext ts,tsx --report-unused-disable-directives --max-warnings 0",
+    "format": "prettier src/**/*.{ts,tsx} --write",
+    "preview": "vite preview",
+    "deploy": "contentful-app-scripts upload --ci --bundle-dir ./build --organization-id ${DEFINITIONS_ORG_ID} --definition-id 3Jt0c7RHcmPsB3mJAQC4iP --token ${CONTENTFUL_CMA_TOKEN}",
+    "deploy:staging": "contentful-app-scripts upload --ci --bundle-dir ./build --organization-id ${TEST_ORG_ID} --definition-id 3zzMa85tjfnkUUJgdBJco3 --token ${TEST_CMA_TOKEN}"
```
[the remark this answers is not in the document]
Comment (zzzarius) [reply apps/adapt-essentials-asset-fields/package.json]: 👍🏻  Fix https://github.com/contentful/marketplace-partner-apps/pull/661/commits/610c04182abc49adb39600a1bdeeaac7c05cf68d

**Review by matthew-gordon — APPROVED** [review]
🚀

**Pull request merged**